contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.spec.js

```diff
@@ -183,6 +181,7 @@ describe('StagingTreePage', () => {
     const getters = cloneDeep(GETTERS);
     getters.currentChannel.hasStagingTree = () => false;
     const wrapper = initWrapper({ getters });
+    wrapper.setData({ isLoading: false });
```

Comment:
Although this is an improvement, I'd still suggest not reaching into the internal methods unless there is no other way for the reasons explained in the first comment regarding this issue. I don't consider it a blocker though.